lib/satismeter/index.js

```diff
@@ -45,20 +45,28 @@ SatisMeter.prototype.loaded = function(){
  */
 
 SatisMeter.prototype.identify = function(identify){
-  var user = {
+  var traits = identify.traits();
+
+  traits.token = this.options.token;
+
+  traits.user = {
     id: identify.userId()
   };
   if (identify.name()) {
-    user.name = identify.name();
+    traits.user.name = identify.name();
   }
   if (identify.email()) {
-    user.email = identify.email();
+    traits.user.email = identify.email();
```

Comment:
I can, but it's already tested in other tests.

https://github.com/satismeter/analytics.js-integrations/blob/master/lib/satismeter/test.js#L76

https://github.com/satismeter/analytics.js-integrations/blob/master/lib/satismeter/test.js#L98

Does it make sense to test it here again?